doc/system_requirements.md

```diff
@@ -400,6 +407,27 @@ Covers:
 
 Needs: dsn
 
+### Driver Management
+
+The Exasol database supports installation of drivers for external data sources through [BucketFS](#bucketfs-access). Installation requires Integrators to follow a convention for storing and registering those drivers.
```

Comment:
is there public documentation on that? In that case you could link it here.

Reply:
Added.